cmd/syncthing/main.go

```diff
@@ -631,7 +649,7 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 		}()
 	}
 
-	perf := cpuBench(3, 150*time.Millisecond, true)
+	perf := api.CpuBench(3, 150*time.Millisecond, true)
```

Comment:
Seems like the wrong this place for this. Scanner maybe?

Reply:
It's also in lib/ur now. The memory and cpu stuff could be moved to their own package, but having them in usage reporting seems sane enough to me.